tools/FormatPythonCode.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Since you have formatted all existing code Python code in this PR, we probably don't need to keep this script in the codebase. We just have to check that modified or added files are correctly formatted in PRs, just like we do with clang-format. What do you think?

Reply:
I think it makes it easy to format the files, even individual ones. It'll also be nice when we want to upgrade to newer versions of yapf because at that point there might be bug fixes or minor changes that mean all code needs reformatting.

Reply:
ok. should such a script also exist for clang-format?

Reply:
Meh

Reply:
I think the difference is if we enforced clang-format we'd want an easy way to format the whole repo. The other part is that there's git-clang-format (I don't know if this also exists for yapf), which makes formatting easier.